src/Admin/AbstractAdmin.php

```diff
@@ -841,14 +841,14 @@ final public function setTemplateRegistry(MutableTemplateRegistryInterface $temp
         $this->templateRegistry = $templateRegistry;
     }
 
-    public function setTemplates(array $templates): void
+    final public function setTemplates(array $templates, bool $override = true): void
```

Comment:
This looks wrong. A *setter* should always set a value, nothing more. You could add a new method `addTemplates` which does the override part

Reply:
Technily it is not `setter` becouse this method merge argument with object property, not override property by argument.
IMO `AddTemplates` is wrong too - you can not add the same template twice.
WDYT about `configureTemplate()` and `configureTemplates()` instead ?